invenio_rdm_records/services/components/pids.py

```diff
@@ -22,68 +22,40 @@
 from ..pids.tasks import register_or_update_pid
 
 
-class PIDsComponent(ServiceComponent):
-    """Service component for PIDs."""
+def _get_optional_doi_transitions(record):
+    """Reusable method to validate optional DOI."""
+    RDM_OPTIONAL_DOI_TRANSITIONS = current_app.config.get(
```

Comment:
Classic comment from me... do we want to use `.get(` here? Or fail if not provided?
If it is OK to be optional, the code should work fine if I set the config to `{}`. Otherwise, let's use `[<key>]` instead :)

Reply:
you are right, I changed that